Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeAdPermissions.ps1

```diff
@@ -99,7 +97,13 @@ Function Get-ExchangeAdPermissions {
     foreach ($domain in $forest.Domains) {
 
         $domainName = $domain.Name
-        $domainDN = $domain.GetDirectoryEntry().distinguishedName
+        try {
+            $domainDN = $domain.GetDirectoryEntry().distinguishedName
```

Comment:
why was this added? curious because we are basically bypassing this domain when it is skipped and could be vulnerable to the CVE.

Reply:
HealthChecker is terminating in a customer environment due to unreachable domain. Check your email.